README.md

```diff
@@ -5,6 +5,8 @@ The RVM Cloud Native Buildpack installs RVM and a Ruby version in an OCI image.
 ## Functionality
 
 1. The RVM CNB installs RVM into its own layer. The version of RVM to be installed can be configured in [buildpack.toml](buildpack.toml).
+1. `Gemfile` and `Gemfile.lock` files must exist in the application directory in order to RVM CNB
+can be able to start DETECTION phase. 
```

Comment:
Please fix the grammar in this sentence: 

The files `Gemfile` and `Gemfile.lock` must exist in the application directory so that the RVM CNB can start the DETECTION phase.